server/command.go

```diff
@@ -67,7 +68,8 @@ const commonHelpText = "\n" +
 	"* `/jira [issue] unassign [issue-key]` - Unassign the Jira issue\n" +
 	"* `/jira [issue] view [issue-key]` - View the details of a specific Jira issue\n" +
 	"* `/jira help` - Launch the Jira plugin command line help syntax\n" +
-	"* `/jira info` - Display information about the current user and the Jira plug-in\n" +
+	"* `/jira info` - Display information about the current user\n" +
```

Comment:
We've usually used `/(cmd) me` to show user info

```suggestion
	"* `/jira me` - Display information about the current user\n" +
```

Reply:
Good suggestion. Fixed in 1a76b8d.